import-scripts/rds_functions.sh

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env bash
+
+# Get current instance class
+rds_current_class() {
+    local id="$1"
+    local profile="$2"
+
+    aws rds describe-db-instances \
+        --db-instance-identifier "$id" \
+        --query 'DBInstances[0].DBInstanceClass' \
+        --output text \
+        --profile "$profile"
+}
+
+# Get current instance status
+rds_current_status() {
+    local id="$1"
+    local profile="$2"
+
+    aws rds describe-db-instances \
+        --db-instance-identifier "$id" \
+        --query 'DBInstances[0].DBInstanceStatus' \
+        --output text \
+        --profile "$profile"
+}
+
+# Start instance
+rds_start() {
+    local id="$1"
+    local profile="$2"
+
+    aws rds start-db-instance \
+        --db-instance-identifier "$id" \
+        --no-cli-pager \
+        --profile "$profile"
+    aws rds wait db-instance-available \
+        --db-instance-identifier "$id" \
+        --profile "$profile"
+}
+
+# Stop instance
+rds_stop() {
+    local id="$1"
+    local profile="$2"
+
+    aws rds stop-db-instance \
+        --db-instance-identifier "$id" \
+        --no-cli-pager \
+        --profile "$profile"
+
+    # The functionality to check whether an RDS instance is stopped doesn't yet exist from the AWS CLI;
+    # we have to emulate it ourselves manually
+    # aws rds wait db-instance-stopped --db-instance-identifier "$id"
+
+    while true; do
```

Comment:
This loop may become infinite if the attempt to stop the instance failed in some way which causes the instance to never stop. Perhaps we should consider a maximum loop count?